subjects/devops/days_and_numbers/README.md

```diff
@@ -0,0 +1,99 @@
+## Days and numbers
+
+### Instructions
+
+Create a file `custom_calendar.py` which will have 2 functions:
+
+- `day_from_number(day_number)`
+- `day_to_number(day)`
+
+Those functions perform conversion between day `index` and day `word` and vice versa:
```

Comment:
"and vice versa" is redundant since "conversion between" is already a two way road.

Reply:
done